tests/components/zha/test_light.py

```diff
@@ -63,6 +67,43 @@
 }
 
 
+@asynctest.mock.patch(
+    "homeassistant.components.zha.light.Light._refresh", new=MagicMock()
+)
+async def test_light_refresh(hass, zigpy_device_mock, zha_device_joined_restored):
+    """Test zha light platform refresh."""
+
+    # create zigpy devices
+    zigpy_device = zigpy_device_mock(LIGHT_ON_OFF)
+    zha_device = await zha_device_joined_restored(zigpy_device)
+    entity_id = await find_entity_id(DOMAIN, zha_device, hass)
+    assert entity_id is not None
+
+    light_entity = hass.data[DOMAIN].get_entity(entity_id)
+    assert light_entity is not None
```

Comment:
We should avoid asserting integration details. They might change in the future.

Reply:
removed